### ISSUE TITLE
Modals not opening in Internet Explorer 11

### ISSUE THREAD

**Issue by JasonDeLierre** [mattlewis92/angular-calendar#759]
**Describe the bug**

Not able to open modals in Internet Explorer 11 when clicking either an event or empty cell in Day view or Week view. Similarly with event in Month view, however, modal does open when clicking an empty cell.

There is no issue in either in Chrome or Firefox though, modals open without an issue on all views.
I'm using Angular Material dialog component to open modals, although not sure if this is the cause of the issue.
There is no error message in the console whatsoever. 

**Minimal reproduction of the problem with instructions**

https://angular-g1kr4d.stackblitz.io/

I know stackblitz may not be supported by Internet Explorer 11.
If there is another way to repro issue, lmk. 
I could also minimize code further if needed. As it is, it replicates my app structure.

**Screenshots**

What the modals should look like when clicking either an event or an empty space.

![screenshot_2018-10-03_07-39-45](https://user-images.githubusercontent.com/33167586/46408315-badf4080-c6df-11e8-8750-c9e81db13ccb.png)

**Versions**

- `@angular/core`: 6.1.9
- `angular-calendar`: 0.24.1
- Browser name and version: Internet Explorer 11

**Comment by mattlewis92** [comment]
Does the `console.log('event', event)` get fired when you click the event? Are you able to open a modal by just a normal button press? Have you made sure all required polyfills for IE11 are included in your app?

**Comment by JasonDeLierre** [comment]
No the console.log('event', event) does not fire.
This makes me think that issue has to do with custom (eventClicked) (segmentClicked) (dayClicked) directives.
I'm not sure why it still works when clicking Month view empty cell though, that being said. 

Yes, I'm able to open a modal through a regular button press.
And yes, I've included all polyfills for IE11.

**Comment by JasonDeLierre** [comment]
So, the 'stackblitz not working in IE11' issue has just been fixed. 
https://github.com/IgniteUI/igniteui-docfx/issues/722

This allowed me to do further testing in IE11, and lo and behold all modals were opening fine. 
I updated all of my dependencies to those in the stackblitz, and my app worked perfectly.

@angular/animations6.1.9
@angular/cdk6.4.7
@angular/common6.1.9
@angular/compiler6.1.9
@angular/core6.1.9
@angular/forms6.1.9
@angular/material6.4.7
@angular/platform-browser6.1.9
@angular/platform-browser-dynamic6.1.9
@angular/router6.1.9
@ngx-translate/core10.0.2
@ngx-translate/http-loader3.0.1
angular-calendar0.24.1
classlist.js1.1.20150312
core-js2.5.7
date-fns1.29.0
hammerjs2.0.8
rxjs6.3.3
rxjs-compat6.3.3
web-animations-js2.3.1
zone.js

Results should be testable in https://stackblitz.com/edit/angular-xtpxkp.
So problem resolved by updating dependencies. Thanks anyway.
